acra-censor/handlers/querycapture_handler.go

```diff
@@ -29,10 +29,10 @@ type QueryCaptureHandler struct {
 	logger               *log.Entry
 }
 
-// QueryInfo describes Query and its status (was forbidden or allowed).
+// QueryInfo describes Query and if it was blocked by external source.
 type QueryInfo struct {
-	RawQuery    string
-	IsForbidden bool
+	RawQuery    string `json:"RawQuery"`
```

Comment:
in json usually used field names in lowercase. so will be better if we will use `raw_query`/`_blocked_by_web_config` names

Reply:
changed